src/pages/Login.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+import styled from "@emotion/styled";
+import Button from "@leafygreen-ui/button";
+import { palette } from "@leafygreen-ui/palette";
+import TextInput from "@leafygreen-ui/text-input";
+import { Location } from "history";
+import { Navigate, useLocation } from "react-router-dom";
+import { size } from "constants/tokens";
+import { useAuthContext } from "context/auth";
+
+const { green } = palette;
+
+const getReferrer = (location: Location): string => {
+  const state = location.state as { referrer?: string };
+  return state?.referrer ?? "/";
+};
+
+export const Login: React.FC = () => {
+  const location = useLocation();
+  const { isAuthenticated, login } = useAuthContext();
+
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  return isAuthenticated ? (
+    <Navigate to={getReferrer(location)} />
+  ) : (
+    <LoginWrapper>
```

Comment:
I think it's probably okay to leave out, since the user can't reach the`/login` page unless they specify it, and it should auto-redirect to Evergreen anyways!